cve_bin_tool/checkers/openssh.py

```diff
@@ -16,7 +17,7 @@ def get_version(lines, filename):
 
     VPkg: openbsd, openssh
     """
-    regex = re.compile(r"OpenSSH_([0-9]+\.[0-9]+[0-9a-z\s]*)")
+    regex = [r"openssh: ([0-9]+\.[0-9]+[0-9a-z\s]*)", r"openssh: ([0-9]+\.[0-9]+)"]
```

Comment:
Please don't remove or replace the regex that starts with`OpenSSH_`.  You've now got two `openssh: ` ones but no `OpenSSH_` one.

Reply:
In addtion... does this openssh: string you're adding actually match anything in a real copy of openssh?  Because when I run strings on /usr/sbin/sshd or /usr/bin/ssh I'm not finding anything that looks like this.  What I am finding are things that look like this:

```
OpenSSH_7.6p1 Ubuntu-4ubuntu0.3
OpenSSH-2.0*,OpenSSH-2.1*,OpenSSH_2.1*,OpenSSH_2.2*
OpenSSH_2.5.0p1*,OpenSSH_2.5.1p1*
OpenSSH_2.5.0*,OpenSSH_2.5.1*,OpenSSH_2.5.2*
OpenSSH_2.*,OpenSSH_3.0*,OpenSSH_3.1*
OpenSSH_2.3.0*
OpenSSH_2.3.*
OpenSSH_2.5.3*
OpenSSH_3.*
OpenSSH_4*
OpenSSH_5*
OpenSSH_6.6.1*
OpenSSH_6.5*,OpenSSH_6.6*
```

Try running `strings /usr/sbin/sshd | grep -i openssh` or `strings /usr/bin/ssh | grep -i openssh` on your system to see what you find, then try running those commands on different files in packages to see what you get. I'll give a detailed example of this in your test below...